bindata/network/ovn-kubernetes/ovnkube-node.yaml

```diff
@@ -233,5 +209,24 @@ spec:
       - name: host-var-lib-cni-networks-ovn-kubernetes
         hostPath:
           path: /var/lib/cni/networks/ovn-k8s-cni-overlay
+      - configMap:
```

Comment:
If these are already included in the ovn configuration files, why do they *also* need to be mounted to disk?

Reply:
ovn-config has a bunch of (ultimately) script variables. "cat" of the file is a simple (quick) way to get a value.
ovnkube-config is mounted and used as the config file for the ovnkube program.
I likely don't understand your question.